src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: 200px;
```

Comment:
The `grid-template-columns` property with a single column size of `200px` is redundant since it does not create multiple columns. You can remove this line if you are not planning to have more than one column before the first breakpoint.